src/main/java/net/rubyeye/xmemcached/aws/AWSElasticCacheClient.java

```diff
@@ -38,6 +42,7 @@
  * @author dennis
  *
```

Comment:
Adds a link to `AutoDiscoveryCacheClient`.

```suggestion
 * @see  net.rubyeye.xmemcached.autodiscovery.AutoDiscoveryCacheClient
 *
```